devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/tooling/maven-wrapper/base/mvnw.tpl.qute

```diff
@@ -212,9 +212,9 @@ else
       echo "Couldn't find .mvn/wrapper/maven-wrapper.jar, downloading it ..."
     fi
     if [ -n "$MVNW_REPOURL" ]; then
-      jarUrl="$MVNW_REPOURL/io/takari/maven-wrapper/{maven.version}/maven-wrapper-{maven.version}.jar"
+      jarUrl="$MVNW_REPOURL/io/takari/maven-wrapper/{maven.wrapper-version}/maven-wrapper-{maven.wrapper-version}.jar"
     else
-      jarUrl="https://repo.maven.apache.org/maven2/io/takari/maven-wrapper/{maven.version}/maven-wrapper-{maven.version}.jar"
+      jarUrl="https://repo.maven.apache.org/maven2/io/takari/maven-wrapper/{maven.wrapper-version}/maven-wrapper-{maven.wrapper-version}.jar"
```

Comment:
So just to be clear: I think it's a VERY bad idea to have copied the Maven Wrapper in our code. It's going to be a nightmare to maintain.

Not related to this PR per se though.

Reply:
I agree with @gsmet here. Ideally the Maven Wrapper plugin should be run to produce these artifacts to avoid people from maintaining this snippet

Reply:
@gastaldi @gsmet if you look closer, you will see that we just provides the scripts (not the actual wrapper) which is downloaded automatically when the user first start the command.

The scripts shouldn't change very often (and are not coupled to any version) so it shouldn't be a burden to maintain them.

Regarding the maven version, we could make it dynamic using the maven version used during the build, I didn't have time to do it and didn't know if it wasn't better to have it coded?

Reply:
@ia3andy I know, it's the wrapper script I was referring to. Since this is auto-generated from the Tanaki plugin I don't see the point why we need to replicate that behavior.

Just my 2c ;)

Reply:
@gastaldi feel free to submit a PR that auto-generate those files in the `src/main/resources/codestarts/quarkus/core/tooling/maven-wrapper/base/` directory, that would be cool indeed. Just be careful not to avoid generating the jar, and if you do it, you could also remove the data in the codestart.yml as the version will already be in the generated file.